src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITestsAntigenTestProfile.swift

```diff
@@ -0,0 +1,141 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+
+class ENAUITestsAntigenTestProfile: XCTestCase {
```

Comment:
Please move this to ENAUITests folder, check for the existing test file (where PCR tests are) if there is already, if not please name it like "ENAUITests_12_AntigenTestProfile"